src/quantum/azext_quantum/operations/job.py

```diff
@@ -88,7 +91,8 @@ def build(cmd, target_id=None, project=None):
     raise AzureInternalError("Failed to compile program.")
 
 
-def _generate_submit_args(program_args, ws, target, token, project, job_name, shots, storage, job_params):
+def _generate_submit_args(program_args, ws, target, token, project, job_name, shots, storage, job_params,
```

Comment:
The implementation on the Q# standalone executables side that is leveraged by the `az quantum` extension (see [qsharp-runtime PR#1015](https://github.com/microsoft/qsharp-runtime/pull/1015)) ended up not having a `--target-capability` argument but instead it gets it from the `TargetCapability` project property.

Having said that, the changes to how the standalone executable is invoked are no longer required.